src/test/resources/Smoke.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
+
+<suite name="TestNG">
+
+    <test name="Example Test" preserve-order="false">
+        <classes>
+
+        <class name = "LoginTests"/>
+            <class name="BaseTest"/>
+                <class name = "ProfileTests"></class>
```

Comment:
Here use class name `"ProfileTests"` but I can't see ProfileTest class add to code. Cannot resolve class 'ProfileTests'